### PR TITLE
[dead] Just playing around

### PR DESCRIPTION
**Pull request by nabobalis**
I am just playing around, its hard to tell what will happen without the CircleCI!

### CODE REVIEW

**Comment by nabobalis** [comment]
@conda-forge-admin, please add noarch: python

**Comment by nabobalis** [comment]
I am wondering if the noarch was a mistake.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,6 +22,7 @@ requirements:
     - coverage
   run:
     - python
+    - enum34  # [py<34]
```
Comment (Cadair) [inline recipe/meta.yaml]: I didn't think conda forge was doing 3.4 builds any more?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
+  name: hypothesis
+  version: {{version}}
 
 source:
-  fn: {{ pkgname }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ pkgname[0] }}/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  fn: hypothesis-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/h/hypothesis/hypothesis-{{version}}.tar.gz
+  md5: cb0ee54fa6b16d5b6c95b7f096f15667
```
Comment (Cadair) [inline recipe/meta.yaml]: I would undo this block of changes, the re-rendered version with templates and sha256 is better and has nothing to do with the error.

**Comment by nabobalis** [comment]
Yeah but I prefer this. Once/if we get it working, we can just pr the fix.

**Comment by nabobalis** [comment]
Nope, couldn't figure out trying around locally with docker or conda build.

File: recipe/meta.yaml (hunk 1 above)
Comment (jochym) [reply recipe/meta.yaml]: That is code from the past.

Comment (Cadair) [reply recipe/meta.yaml]: is it not still relevant for the python 2.7 build?

Comment (nabobalis) [reply recipe/meta.yaml]: Is it not needed anymore? Hypothesis seems to require it for py27 in master.

Comment (nabobalis) [reply recipe/meta.yaml]: Probably best move over to the newly opened PR.

Comment (jochym) [reply recipe/meta.yaml]: Exactly - I believe it is required for 2.7. So it is kind of from the past - but still required ;)

**Pull request closed without merging**